core/src/main/java/com/datastax/oss/driver/internal/core/auth/RuntimePlainTextAuthProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright DataStax, Inc.
```

Comment:
This Could also be folded into the normal PlainTextAuthProvider if appropriate. That's what I did initially, but with the introduction of the PlainTextAuthProviderBase, it seemed appropriate to have a separate class.